backend/libraries/types/can.did

```diff
@@ -1206,4 +1225,28 @@ type MessageIndexRange =
     record {
         start: MessageIndex;
         end: MessageIndex;
-    };
\ No newline at end of file
+    };
+
+type PrizeContentInitial =
+    record {
+        prizes: vec Tokens;
+        token: Cryptocurrency;
+        end_date: TimestampMillis;
+        caption: opt text;
+    };
+
+type PrizeContent =
+    record {
+        prizes_remaining: nat32;
+        winners: vec UserId;
+        token: Cryptocurrency;
+        end_date: TimestampMillis;
+        caption: opt text;
+    };
+
+type PrizeWinnerContent =
+    record {
+        token: Cryptocurrency;
```

Comment:
Since this is a message sent to the group, don't we also need to include the recipient / winner's UserId in the `PrizeWinnerContent`?

Reply:
Yes you're right